ci-operator/step-registry/ipi/conf/aws/usage-info/ipi-conf-aws-usage-info-commands.sh

```diff
@@ -44,6 +35,22 @@ usage-pull-request ${pull_number}
 usage-ci-type ${ci_type}
 EOF
 
+# for hypershift mgmt cluster, store one more cluster info file for the hosted cluster
+if [[ "${USAGE_CLUSTER_TYPE}" == "hypershift-mgmt" ]]; then
+  hosted_cluster_file="${SHARED_DIR}/hosted_cluster_info_file"
+  cp "${cluster_info_file}" "${hosted_cluster_file}"
+  sed -ie 's|usage-cluster-type.*|usage-cluster-type hypershift-hosted|' "${hosted_cluster_file}"
```

Comment:
Personally I think we should update `usage-cluster-type` field of `${hosted_cluster_file}` in hypershift-aws-create-chain.yaml when the hosted cluster is really being installed.

Reply:
use this way, we can add the tag for the hypershift-hosted automatically when hypershift-mgmt is set.
In fact I don't want to edit this file in multiple steps. here it prepares all of the the hosted info together and then we can use it in other place as read-only to make the logic easier to maintain

Reply:
Okay, if you prefer to prepare the hosted cluster info in the hacking way in advance, I am okay with it.

Reply:
thanks! I will take a look at the running status later and if it is not available, I will update it to improve the way.